packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -7772,6 +7773,116 @@ QUnit.module('label integration', () => {
 });
 
 QUnit.module('accessibility', () => {
+    QUnit.test('multitag should have correct aria attributes', function(assert) {
+        const sampleData = ['First Item', 'Second Item', 'Third Item', 'Fourth Item'];
+        const $tagBox = $('#tagBox').dxTagBox({
+            items: sampleData,
+            value: sampleData,
+            maxDisplayedTags: 2,
+            showMultiTagOnly: false,
+        });
+        const $parentTagContainerIds = $tagBox.attr('aria-labelledby').split(' ');
+        const $multiTagContainer = $tagBox.find(`.${TAGBOX_MULTI_TAG_CLASS}`);
+        const totalSelectedItems = $tagBox.dxTagBox('instance').option().selectedItems.length;
+
+        assert.strictEqual($parentTagContainerIds.includes($multiTagContainer.attr('id')), true, 'aria-labelledby attribute contains multitag id');
+        assert.strictEqual($multiTagContainer.attr('role'), 'button', 'role attribute is correct');
+        assert.strictEqual($multiTagContainer.attr('aria-label'), `${totalSelectedItems - 1} more`, 'aria-label attribute is correct');
+        assert.strictEqual($multiTagContainer.attr('aria-roledescription'), messageLocalization.format('dxTagBox-tagRoleDescription'), 'aria-roledescription attribute is correct');
+    });
+
+    QUnit.test('root element should have aria-labelledby attribute based from multitag id', function(assert) {
+        const sampleData = ['First Item', 'Second Item', 'Third Item', 'Fourth Item'];
+        const $tagBox = $('#tagBox').dxTagBox({
+            items: sampleData,
+            value: sampleData,
+            maxDisplayedTags: 2,
+            showMultiTagOnly: false,
+        });
+        const $multiTagContainer = $tagBox.find(`.${TAGBOX_MULTI_TAG_CLASS}`);
+        const $parentTagContainerIds = $tagBox.attr('aria-labelledby').split(' ');
+
+        assert.strictEqual($parentTagContainerIds.length, 2, 'root element aria-labelledby attribute contains two ids');
+        assert.strictEqual($parentTagContainerIds.includes($multiTagContainer.attr('id')), true, 'aria-labelledby attribute contains multitag id');
+    });
+
+    QUnit.test('check if tagBox is showing the right count of tags correctly if showMultiTagOnly=true and maxDisplay tags is 2', function(assert) {
```

Comment:
According to our style guide it's better to make test name answering on a question "how should it work?"
```suggestion
    QUnit.test('tagBox should show correct count of tags if showMultiTagOnly=true and maxDisplay tags is 2', function(assert) {
```
This approach makes tests names more short and clear. 
Please update other tests names as well.